roles/config-nexus/tasks/wait-for-nexus.yml

```diff
@@ -8,6 +8,7 @@
     user: "{{ nexus_user }}"
     password: "{{ nexus_password }}"
     force_basic_auth: yes
+    validate_certs: false
```

Comment:
Can we please make this configurable and default it to `true`.

Reply:
Done